python/build-wheel-macos.sh

```diff
@@ -6,79 +6,66 @@ set -e
 # Show explicitly which commands are currently running.
```

Comment:
This is actually moved from build-wheel-macos-arm64.sh

I think we could unify the two for building macos wheels.